File: Data Structures & Algorithms/majority-element/submission-0.java
class Solution {
    public int majorityElement(int[] arr) {
        int maj = arr[0];
        int count=0;

        for(int i=0;i<arr.length;i++){
            if(count==0){
                maj = arr[i];
                count =1;
            }
            else if(arr[i]==maj){
                count++;
            }
            else{
                count--;
            }
        }

        return maj;
        
    }
}

File: Data Structures & Algorithms/height-checker/submission-0.py
class Solution:
    def heightChecker(self, arr: List[int]) -> int:
        new_arr = sorted(arr)
        count= 0
        for i in range(len(new_arr)):
            if arr[i]==new_arr[i]:
                continue
            else:
                count+=1
        
        return count

